packages/router/__tests__/router-test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/valid-title */
```

Comment:
This rule doesn't like spaces at the start or end of tests names, which makes sense for normal test names, but disable for this file due to our illustrative tests:

```
      describe(`
        A) fetch POST |----|-----O
        B) nav GET      |-----O
      `, () => {
```